project1/styles.css

```diff
@@ -0,0 +1,60 @@
+h1 {
+    font-size: 70px;
+    text-align: center;
+    background-color: darkgrey;
+    font-family: 'IM Fell French Canon SC', serif;
+    color: #034247;
+
+}
+
+h2 {
+    font-size: 20px;
+    text-align: center;
+    background-color: darkgrey;
+}
+
+body {
+    margin: 0 auto;
+    max-width: 800px;
+    background-color: darkgrey;
+    font-family: 'Abel', sans-serif;
+    color: black;
+}
+
+hr {
+    width: 100%;
+    height: 2px;
+    background: #fff;
+}
+
+div {
+    text-align: center;
+    font-size: 80px;
```

Comment:
This `font-size` rule also gets ovewritten by the styles for the "star" class and your `p` element, so it can go, too
```suggestion
```